src/index.html

```diff
@@ -1,19 +1,181 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width,
-          user-scalable=no,
-          initial-scale=1.0,
-          maximum-scale=1.0,
-          minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-
-<script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width,
+            user-scalable=no,
+            initial-scale=1.0,
+            maximum-scale=1.0,
+            minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>layout_catalog</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <section class="body-wrapper">
+
+      <header class="header-of-site">
+        <a href="#" class="logo-inside"><img class="logo-in-header" src="images/logo.png" alt="logo"></a>
+        <nav class="nav-links">
+          <ul class="list-of-links">
+            <li class="li-item"><a class="link Apple is-active" href="#">apple</a></li>
+            <li class="li-item"><a class="link Samsung" href="#">samsung</a></li>
+            <li class="li-item"><a class="link phones" href="#">смартфоны и телефоны</a></li>
+            <li class="li-item"><a class="link PC" data-qa="nav-hover" href="#">ноутбуки и компьютеры</a></li>
+            <li class="li-item"><a class="link gadgets" href="#">гаджеты</a></li>
+            <li class="li-item"><a class="link Tabs" href="#">планшеты</a></li>
+            <li class="li-item"><a class="link photo" href="#">фото</a></li>
+            <li class="li-item"><a class="link video" href="#">видео</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <main>
+
+        <div class="main-wrapper">
+
+          <div class="entire-card" data-qa="card">
+            <img class="logo-in-card" src="images/imac.jpeg" alt="logo">
+            <a href="#" class="product-name product-text">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</a>
```

Comment:
Классы должны быть собраны в одну логику
```suggestion
            <a href="#" class="product-card__name product-card__text">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</a>
```